esp/logging/logginglib/loggingagentbase.cpp

```diff
@@ -154,174 +154,172 @@ void CLogContentFilter::filterLogContentTree(StringArray& filters, IPropertyTree
 
 IEspUpdateLogRequestWrap* CLogContentFilter::filterLogContent(IEspUpdateLogRequestWrap* req)
 {
-    Owned<IPropertyTree> scriptValues = req->getScriptValuesTree();
-    const char* logContent = req->getUpdateLogRequest();
+    bool noFilter = logBackEndReq && logBackEndResp && (groupFilters.length() < 1);
     Owned<IPropertyTree> logRequestTree = req->getLogRequestTree();
-    Owned<IPropertyTree> updateLogRequestTree = createPTree("UpdateLogRequest");
+    const char* logContent = req->getUpdateLogRequest();
+    if (logRequestTree || !isEmptyString(logContent))
+    {
+        if (noFilter)
+            return LINK(req);
```

Comment:
This has the potential to break any deployment that has already discovered the extra element level and adapted to it. The change is probably worthwhile, but may warrant a red book mention in case anyone's logging stops working.

Reply:
On second thought, it looks like the change that introduced this behavior has only been included in nightly builds. If this is true, this comment can be disregarded.